Add printing entitlement to Minibrowser

nmoucht opened the pull request:
#### 8dfe17d903738a524a2369df7ddf723d7b75db18
<pre>
Add printing entitlement to Minibrowser
<a href="https://bugs.webkit.org/show_bug.cgi?id=248687">https://bugs.webkit.org/show_bug.cgi?id=248687</a>
rdar://102918496

Reviewed by Simon Fraser.

Add printing entitlement to Minibrowser. Also set
CODE_SIGN_ALLOW_ENTITLEMENTS_MODIFICATION as I was getting
build failures complaining about the entitlement file changing.

* Tools/MiniBrowser/Configurations/Base.xcconfig:
* Tools/MiniBrowser/MiniBrowser.entitlements:

Canonical link: <a href="https://commits.webkit.org/257328@main">https://commits.webkit.org/257328@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/9c01890995e088602ed3cdf41fef55cdb93f6e7f

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/98539 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/7749 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/31666 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/107961 "Built successfully") | [✅ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/168229 "Built successfully and passed tests") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/102488 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/8259 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/85132 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/91078 "Built successfully") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/12/builds/105496 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/104210 "Passed tests") | [❌ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/6265 "Found 2 new test failures: css3/supports-dom-api.html, fast/text/text-edge-no-half-leading-simple.html (failure)") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/89822 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/33253 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/88083 "Passed tests") | [❌ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/21181 "Found 2 new test failures: css3/supports-dom-api.html, fast/text/text-edge-no-half-leading-simple.html (failure)") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/76193 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/1684 "Built successfully") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/70/builds/22710 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/1600 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/60/builds/45208 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/5044 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/6533 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/42134 "Passed tests") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/2982 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

smfr commented on Tools/MiniBrowser/MiniBrowser.entitlements:
```diff
@@ -20,16 +26,12 @@
 	<array>
-		<string>(allow mach-issue-extension (require-all (extension-class &quot;com.apple.webkit.extension.mach&quot;)))</string>
-		<string>(allow iokit-issue-extension (require-all (extension-class &quot;com.apple.webkit.extension.iokit&quot;)))</string>
+		<string>(allow mach-issue-extension (require-all (extension-class "com.apple.webkit.extension.mach")))</string>
```
Are the quoting changes OK?

nmoucht committed:
Add printing entitlement to Minibrowser
https://bugs.webkit.org/show_bug.cgi?id=248687
rdar://102918496

Reviewed by Simon Fraser.

Add printing entitlement to Minibrowser. Also set
CODE_SIGN_ALLOW_ENTITLEMENTS_MODIFICATION as I was getting
build failures complaining about the entitlement file changing.

* Tools/MiniBrowser/Configurations/Base.xcconfig:
* Tools/MiniBrowser/MiniBrowser.entitlements:

Canonical link: https://commits.webkit.org/257328@main
